test/sqllogictest/object_ownership.slt

```diff
@@ -1258,6 +1258,21 @@ ALTER INDEX mz_internal.mz_show_databases_ind OWNER TO mz_system
 ----
 db error: ERROR: cannot alter item mz_internal.mz_show_databases_ind because it is required by the database system
 
+# Superusers can alter the owner to a role that they are not a member of
+
+statement ok
+CREATE TABLE t ()
+
+simple conn=mz_system,user=mz_system
```

Comment:
Could also check if it works when another role created it, not mz_system itself.

Reply:
Done